contrib/appveyor-msys2-qt5-static.bat

```diff
@@ -0,0 +1,52 @@
+set BITS=""
+set ARCH=""
+if "%1%"=="" (
+  if "%MBITS%" == "" (set BITS=64) else (set BITS=%MBITS%)
```

Comment:
Can you keep the default to 32 bits? since all the other scripts do it too.

Reply:
Done